ext/websocket/01_websocket.js

```diff
@@ -508,7 +506,7 @@ class WebSocket extends EventTarget {
           if (prevState === OPEN) {
             try {
               await core.opAsync(
-                this[_role] === SERVER ? "op_server_ws_close" : "op_ws_close",
+                "op_ws_close",
```

Comment:
Why are you going back to having a single op for both client and server?

Reply:
Because the client and server apis are the same with `fastwebsockets::Role` being `Client` or `Server`